query-service-impl/src/main/java/org/hypertrace/core/query/service/attribubteexpression/AttributeExpressionNormalizationTransformation.java

```diff
@@ -0,0 +1,41 @@
+package org.hypertrace.core.query.service.attribubteexpression;
+
+import io.reactivex.rxjava3.core.Single;
+import lombok.extern.slf4j.Slf4j;
+import org.hypertrace.core.query.service.AbstractQueryTransformation;
+import org.hypertrace.core.query.service.QueryRequestUtil;
+import org.hypertrace.core.query.service.api.ColumnIdentifier;
+import org.hypertrace.core.query.service.api.Expression;
+import org.slf4j.Logger;
+
+@Slf4j
+final class AttributeExpressionNormalizationTransformation extends AbstractQueryTransformation {
+
+  @Override
+  public int getPriority() {
+    // Run before default transformations
+    return 1;
+  }
+
+  @Override
+  protected Logger getLogger() {
+    return log;
+  }
+
+  @Override
+  protected Single<Expression> transformColumnIdentifier(ColumnIdentifier columnIdentifier) {
+    Expression.Builder expressionBuilder =
+        QueryRequestUtil.createSimpleAttributeExpression(columnIdentifier.getColumnName());
```

Comment:
What will happen with column identifier with `CONTAIN_KEY_VALUE` operator? So, after this change, would that be broken or taken care of in order deployment?

Reply:
I'd expect it to translate same as anything else - something like:
```json
{
      "lhs": {
        "attributeExpression": {
          "attributeId": "EVENT.spanTags"
        }
      },
      "operator": "CONTAINS_KEY_VALUE",
      "rhs": {
        "literal": {
          "value": {
            "stringArray": ["span.kind", "client"]
          }
        }
      }
    }
```

If that doesn't work (we can and should add tests for it), then we'd be broken with or without this change, because we don't expect column identifiers to be coming at all once the gateway change is merged.

Reply:
To elaborate on my current understanding, I thought the support we didn't add WRT `CONTAIN_KEY_VALUE ` was for it to take an attribute expression with subpath on the LHS and a single value on the RHS. Likewise, we didn't add support for `[NOT_]CONTAINS_KEY` with a subpath'd expression and no args. Given that, any arriving expression using a column identifier necessarily has to use the old RHS format so should translate identically.

Reply:
> I'd expect it to translate same as anything else - something like:

Currently unit test is not there for such case as it was being deprecated. Can add it in my PR.

Reply:
Added in this PR.

Reply:
Yes, For this - https://github.com/hypertrace/query-service/pull/129/files#r775325504,  I re-looked the code, and it is taken care of here in method `convertExpressionToMapLiterals` for both `string_array` and for `string`.

So, it looks fine to me.